.github/workflows/cmake-build-example-win.yml

```
@@ -0,0 +1,48 @@
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-example-win:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        build_type: [Release, Debug]
+    steps:
+    - name: Cache Conan
+      uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)